Okta.Xamarin/Okta.Xamarin/Services/MockDataStore.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Okta.Xamarin.Models;
+
+namespace Okta.Xamarin.Services
+{
+    public class MockDataStore : IDataStore<Item>
```

Comment:
Is there any reason for having testing models in the Xamarin project? Are you planning to move these to a separate test project in the future?

Reply:
There is additional refactoring that needs to be done to better organize the project structure for future extensibility; this file along with others should be moved as you suggest (looking closer, this comment also applies here: https://github.com/okta/okta-oidc-xamarin/pull/16#discussion_r496144379)